Everyday/Everyday/Screens/AuthScreen/AuthScreenVC.swift

```diff
@@ -0,0 +1,89 @@
+//
+//  AuthScreenVC.swift
+//  Everyday
+//
+//  Created by Михаил on 06.11.2023.
+//
+
+import UIKit
+
+class AuthScreenVC: UIViewController {
+    
+    private let imageArea = UIImageView()
+    private let textLabel = UILabel()
+    private let everydayLabel = UILabel()
+    private let regButton = UIButton()
+    private let loginButton = UIButton()
+    private var textStackView = UIStackView()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        view.backgroundColor = UIColor(red: 6 / 255, green: 38 / 255, blue: 55 / 255, alpha: 1)
+        
+        view.addSubview(imageArea)
+        view.addSubview(textStackView)
+        view.addSubview(regButton)
+        view.addSubview(loginButton)
+    }
+    
+    override func viewDidLayoutSubviews() {
+        super.viewDidLayoutSubviews()
+        
+        if let logotype = UIImage(named: "logo.png") {
+            imageArea.image = logotype
+        }
+        
+        imageArea.translatesAutoresizingMaskIntoConstraints = false
+        imageArea.widthAnchor.constraint(equalToConstant: UIScreen.main.bounds.size.width * 0.75).isActive = true
+        imageArea.heightAnchor.constraint(equalToConstant: UIScreen.main.bounds.size.width * 0.75).isActive = true
+        
+        imageArea.centerXAnchor.constraint(equalTo: view.safeAreaLayoutGuide.centerXAnchor).isActive = true
+        imageArea.centerYAnchor.constraint(equalTo: view.safeAreaLayoutGuide.centerYAnchor, constant: -0.18 * view.bounds.height).isActive = true
```

Comment:
Предлагаю вот такой вариант – приятнее выглядит, по-моему:
```suggestion
        NSLayoutConstraint.activate([
            imageArea.widthAnchor.constraint(equalToConstant: UIScreen.main.bounds.size.width * 0.75),
            imageArea.heightAnchor.constraint(equalToConstant: UIScreen.main.bounds.size.width * 0.75),
            imageArea.centerXAnchor.constraint(equalTo: view.safeAreaLayoutGuide.centerXAnchor),
            imageArea.centerYAnchor.constraint(equalTo: view.safeAreaLayoutGuide.centerYAnchor, constant: -0.18 * view.bounds.height)
        ])
```